lua/core/mappings.lua

```diff
@@ -196,7 +196,7 @@ M.lspconfig = {
 
       ["<leader>fm"] = {
          function()
-            vim.lsp.buf.formatting()
+            vim.lsp.buf.format({async=true})
```

Comment:
@AlexanderHOtt `vim.lsp.buf.format` does not exists in NVIM v0.7.0, build type "Release", so this cannot be used for now.